content/python/concepts/strings/terms/splitlines/splitlines.md

```diff
@@ -0,0 +1,84 @@
+---
+Title: '.splitlines()' 
+Description: 'Used to split a multi-line string into a list of lines.'
+Subjects: 
+  - 'Data Science'
+  - 'Web Development'
+  - 'Computer Science'
+Tags: 
+  - 'Strings'
+  - 'Methods'
+  - 'Formatting'
+  - 'Lists'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+**`.splitlines()`** is a built-in string method in Python that is used to split a multi-line string into a list of lines. It recognizes different newline characters such as `\n`, `\r`, or `\r\n` and splits the string at those points. The method returns a list of strings, each corresponding to a line in the original multi-line string.
+
+## Syntax
+
+```pseudo
+string.splitlines(keepends=False)
+```
+
+- `string`: This is the string on which to apply the `.splitlines()` method.
+- `keepends` (optional): This is a boolean parameter. If `True`, the line break characters are included in the resulting lines. If `False` (the default), the line break characters are excluded.
+
+## Example
+
+In this example, `.splitlines(keepends=True)` is used to include the line break characters in the resulting lines.
+
+```py
+multi_line_string = "This is line 1.\nThis is line 2.\nThis is line 3."
+
+lines_with_breaks = multi_line_string.splitlines(keepends=True)
+
+for line in lines_with_breaks:
+    print(line)
+```
+
```

Comment:
I think it may be helpful to also include a print statement of the list, to better illustrate the difference with including the endings or not
print(lines_with_breaks)

Reply:
also be sure to use 2 spaces for indentations not tabs